### PR TITLE
Move tests app

### PR DESCRIPTION
**Pull request by loic425**

### CODE REVIEW

**Review by jakubtobiasz — APPROVED** [review]
Similarly to ResourceBundle, we can move the `spec` dir to the root directory too.

File: ecs.php [hunk 1]
```diff
@@ -9,25 +9,28 @@
...
-        ->set(HeaderCommentFixer::class)->call('configure', [[
-            'location' => 'after_open',
-            'header' =>
-'This file is part of the Sylius package.
+    $ecsConfig->ruleWithConfiguration(HeaderCommentFixer::class, [
+        'location' => 'after_open',
+        'header' =>
+            'This file is part of the Sylius package.
 
 (c) Paweł Jędrzejewski
```
Comment (lchrusciel) [inline ecs.php:25]: @Rafikooo you should update licences in technical libs as well, shouldn't you?

**Review by lchrusciel — APPROVED** [review]

**Comment by lchrusciel** [comment]
Thank you, @loic425!

File: ecs.php (hunk 1 above)
Comment (loic425) [reply ecs.php:25]: yes, Lukasz, it will be another PR.

Comment (Rafikooo) [reply ecs.php:25]: @lchrusciel, sure! 🚀

**Pull request merged**